### PR TITLE
feat(few-shot): add a few-shot example for #1

### PR DESCRIPTION
**Pull request by ganler**

### CODE REVIEW

File: prompts/self-ossinstruct-fewshot.txt [hunk 1]
```diff
@@ -681,4 +681,83 @@ assert create_folds(range(12), 2) == [range(0, 6), range(6, 12)]
...
+        T = To * (1 - 6.87559e-6 * h)
+    return T
+
+[Property]
+category: code generation (function implementation)
+language: Python
+concepts: mathematical modeling, environmental physics, parameter default values
+
+[Instruction]
+Write a Python function that converts altitude to temperature using the standard atmosphere model. The function should also adjust for variations in barometric pressure given by the parameter mmHg.
```
Comment (UniverseFly) [inline prompts/self-ossinstruct-fewshot.txt:709]: The instruction loses context.  "the standard atmosphere model" is not explained and terms like "barometric pressure", "mmHg" are strange. Try to make it more declarative, logical, and understandable by humans, incorporating more unique coding features.

Comment (ganler) [reply prompts/self-ossinstruct-fewshot.txt:709]: I thought The Standard Atmosphere was common knowledge: https://engineering.purdue.edu/~andrisan/Courses/AAE490A_S2002/Atmosphere.pdf

Or do you think more explanation is needed regardless of common knowledge? 🤔

File: prompts/self-ossinstruct-fewshot.txt [hunk 2]
```diff
@@ -681,4 +681,83 @@ assert create_folds(range(12), 2) == [range(0, 6), range(6, 12)]
...
+    if h > 36089:
+        T = 389.99
+    else:
+        T = To * (1 - 6.87559e-6 * h)
+    return T
+
+[Property]
+category: code generation (function implementation)
+language: Python
+concepts: mathematical modeling, environmental physics, parameter default values
```
Comment (UniverseFly) [inline prompts/self-ossinstruct-fewshot.txt:706]: mathematical modeling and environmental physics are too vague concepts. Try to relate to the logic like formula calculation / logarithm using..

File: prompts/self-ossinstruct-fewshot.txt [hunk 3]
```diff
@@ -681,4 +681,83 @@ assert create_folds(range(12), 2) == [range(0, 6), range(6, 12)]
...
+
+[Instruction]
+Write a Python function that converts altitude to temperature using the standard atmosphere model. The function should also adjust for variations in barometric pressure given by the parameter mmHg.
+
+[Response]
+[Reasoning]
+To compute the temperature at a given altitude, considering the barometric pressure, we can use the standard atmospheric model modified for non-standard pressures. The function altitude_to_temperature calculates the temperature at a specified altitude h, adjusting the altitude if the barometric pressure mmHg differs from the standard 29.92 inches of Hg. The temperature calculation depends on whether the altitude is above or below 36,089 feet.
```
Comment (UniverseFly) [inline prompts/self-ossinstruct-fewshot.txt:713]: 29.92 cannot be derived from the instruction, so the response is not aligned

**Comment by UniverseFly** [comment]
We have a formula calculation example already, so let's pick a different instrucion

**Pull request closed without merging**